Patch-Source: https://lore.kernel.org/netfilter-devel/D711RJX8FZM8.1ZZRJ5PYBRMID@pwned.life/
---
From a81896ac8c0fcc73ee52603748f876375906cead Mon Sep 17 00:00:00 2001
From: fossdd <fossdd@pwned.life>
Date: Mon, 13 Jan 2025 16:08:34 +0100
Subject: [PATCH] configure: Avoid addition assignment operators

For compatability with other /bin/sh like busybox ash, since they don't
support the addition assignment operators (+=) and otherwise fails with:

	./configure: line 14174: regular_CFLAGS+= -D__UAPI_DEF_ETHHDR=0: not found

Signed-off-by: fossdd <fossdd@pwned.life>
---
 configure.ac | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index 2d38a4d4..0106b316 100644
--- a/configure.ac
+++ b/configure.ac
@@ -202,8 +202,8 @@ fi;
 pkgdatadir='${datadir}/xtables';
 
 if test "x$enable_profiling" = "xyes"; then
-	regular_CFLAGS+=" -fprofile-arcs -ftest-coverage"
-	regular_LDFLAGS+=" -lgcov --coverage"
+	regular_CFLAGS="$regular_CFLAGS -fprofile-arcs -ftest-coverage"
+	regular_LDFLAGS="$regular_LDFLAGS -lgcov --coverage"
 fi
 
 AC_MSG_CHECKING([whether the build is using musl-libc])
@@ -222,7 +222,7 @@ AC_COMPILE_IFELSE(
 AC_MSG_RESULT([${enable_musl_build}])
 
 if test "x$enable_musl_build" = "xyes"; then
-	regular_CFLAGS+=" -D__UAPI_DEF_ETHHDR=0"
+	regular_CFLAGS="$regular_CFLAGS -D__UAPI_DEF_ETHHDR=0"
 fi
 
 define([EXPAND_VARIABLE],
-- 
2.48.0

